src/server/services/readMetasysData.js

```diff
@@ -10,23 +10,23 @@ const Meter = require('./../models/Meter');
 async function readMetasysData(filePath) {
```

Comment:
Thanks for cleaning this up a bit while you were doing other things. It slipped through linting somehow. I see there are a few other linting errors in here, but they aren't your code so it's no big deal.